src/test/init/apphosting/index.spec.ts

```diff
@@ -66,10 +67,10 @@ describe("operationsConverter", () => {
         repository: cloudBuildConnRepo.name,
         rootDirectory: "/",
       },
-      labels: {},
+      labels: deploymentTool.labels(),
     };
 
-    it("should createBackend", async () => {
+    it("should createBackend with rollout policy", async () => {
```

Comment:
Nothing here tests to make sure the rollout policy was called. Please add the appropriate logic here to test for that

Reply:
I may need to revert the label. I originally wrote the test to exercise this but realized it was for a smaller utility function and the larger flow is completely untested. I can either:

1. Move creation of the rollout policy into a utility function and test it. This would be a trivial whitebox test that is unlikely to catch anything though.
2. Add a new test that covers the overall flow of the entiere init process.
3. Cry that init functions are generally sparsely tested in this codebase.

You make the call and I'll follow.

Reply:
Ah, yeah, this isn't actually testing `doSetup`. Probably just revert the test name then. init function coverage is kinda a known weak point.